suite-common/wallet-core/src/accounts/accountsReducer.ts

```diff
@@ -288,3 +285,15 @@ export const selectIsAccountWithRatesByKey = (
 
     return !!rates.find(rate => rate.symbol === account.symbol);
 };
+
+export const selectIsAccountsListEmpty = (state: AccountsRootState) =>
```

Comment:
I know, but since it was like this before this PR and you're still not allowed to switch between them, we should handle this separately and not bloat this PR. As empty state, this works and we will need to do this but not right now. Please create an issue and we can plan this properly.